INSTALL.md

```diff
@@ -83,13 +83,12 @@ your `.zshrc`.
     git clone https://github.com/zsh-users/zsh-syntax-highlighting.git ${ZSH_CUSTOM:-~/.oh-my-zsh/custom}/plugins/zsh-syntax-highlighting
     ```
 
-2. Activate the plugin in `~/.zshrc`:
+2. Activate the plugin:
 
     ```zsh
-    plugins=( [plugins...] zsh-syntax-highlighting)
+    omz plugin enable zsh-syntax-highlighting
```

Comment:
Is this syntax supported by all oh-my-zsh versions run by users who'll be reading this file?